src/periphery/SablierMerkleLL.sol

```diff
@@ -36,6 +36,9 @@ contract SablierMerkleLL is
     /// @inheritdoc ISablierMerkleLL
     MerkleLL.Schedule public override schedule;
 
+    /// @inheritdoc ISablierMerkleLL
+    LockupLinear.UnlockAmounts public override unlockAmounts;
```

Comment:
Instead of creating is as a separate variable, how about merging it into `schedule`. The name `schedule` is very generic and can include amounts as well.

Reply:
hmm, not sure what to say, will think more

Reply:
i would like to keep the current version to have two different structs 1. for time related variables 2. amount related amounts

Reply:
1. https://chatgpt.com/share/67361585-525c-8000-a6fc-dfe1c0e46207
2. https://ltse.com/insights/what-is-a-standard-vesting-schedule

As per above two links, schedule can have both vesting amounts and durations.

Reply:
https://github.com/sablier-labs/v2-core/pull/1088